libbeat/scripts/generate_fields_docs.py

```diff
@@ -90,26 +89,28 @@ def fields_to_asciidoc(input, output, beat):
 
     sections = docs["sections"]
 
+    # Add default section
+    sections["beat"] = "Beat"
+
     # Check if sections is define
     if sections is None:
         print "No sections are defined in fields.yml. fields.asciidoc cannot be generated."
         return
 
-    for doc, _ in sections:
-        output.write("* <<exported-fields-{}>>\n".format(doc))
-    output.write("\n")
 
-    for value in sections:
+    for key in sorted(sections):
+        output.write("* <<exported-fields-{}>>\n".format(key))
+    output.write("\n")
 
-        doc = value[0]
-        name = value[1]
+    for key in sorted(sections):
```

Comment:
@ruflin @tsg In general, I prefer sorting alphabetically unless there is a compelling reason to do otherwise. Info that is sorted alphabetically is typically easier for users to retrieve.